code/modules/pool/pool_items.dm

```diff
@@ -0,0 +1,93 @@
+/obj/item/proc/check_float(mob/living/holder)
+	return FALSE
```

Comment:
it seems a bit hacky to have this proc for every single item when only 2 items will ever return true. Wouldn't it be better to have a subtype check (`/obj/item/twohanded/required/pool`) and have the proc for those items only?